portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,72 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload ="getComments()">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Deron Gentle's Portfolio</h1>
+
+      <img src="/Images/gradpic.jpg" alt ="graduation picture" width="384" height="512">
+
+      <p>I'm currently studying computer science at the University of Central Florida and interning
+         at Google. Once I graduate college I plan to get a job as a software engineer.
+      </p>
+
+      <a href="https://linkedin.com/in/dgent-5445a118b">LinkedIn<a/>
+      <a href="https://github.com/dag13">Github<a/>
+
+      <h2>Projects</h2>
+
+      <p><u>Text Prediction Algorithm</u>  Date: April 2019
+      <br>Written in C, this algorithm utilizes the trie data structure for efficient text 
+          prediction as well as fast insertion and search of words.
+      </p>
+
+      <p><u>Lonely Party Arrays</u> Date: March 2019
+      <br>An advanced data structure written in Java that automattically allocates and deallocates
+          arrays when needed. This helps to reduces tje unnecessary space that standard arrays take
```

Comment:
"tje" spelling mistake?

Reply:
Sorry I saw these problems after I pushed the changes yesterday. My next pull request will reflect these changes.